apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -1052,6 +1046,18 @@ id(BridgeType, BridgeName, ConnectorName) ->
 source_id(BridgeType, BridgeName, ConnectorName) ->
     id_with_root_name(?ROOT_KEY_SOURCES, BridgeType, BridgeName, ConnectorName).
 
+get_resource_ids(ConfRootKey, Type, Name) ->
+    try
+        ChannelResId = id_with_root_name(ConfRootKey, Type, Name),
+        ConnResId = extract_connector_id_from_bridge_v2_id(ChannelResId),
+        {ok, {ConnResId, ChannelResId}}
+    catch
+        throw:Reason ->
+            {error, Reason};
+        error:{error, Reason} ->
```

Comment:
`extract_connector_id_from_bridge_v2_id` should `throw` ?

Reply:
https://github.com/emqx/emqx/blob/23f18a3daccf9975664c4a105d56e0332361db79/apps/emqx_bridge/src/emqx_bridge_v2.erl#L1060-L1067